component/mimir/rules/debug.go

```diff
@@ -0,0 +1,64 @@
+package rules
+
+import "fmt"
+
+type DebugInfo struct {
+	Error               string                   `river:"error,attr,optional"`
+	PrometheusRules     []DebugK8sPrometheusRule `river:"prometheusRules,attr,optional"`
+	MimirRuleNamespaces []DebugMimirNamespace    `river:"mimirRuleNamespaces,attr,optional"`
```

Comment:
These should probably be blocks (not attrs) so they get dedicated sections in the UI rather than all being shoved into the same table. Also, when changing them to blocks, make the names singular rather than plural.